aiohttp/client.py

```diff
@@ -178,6 +178,8 @@ class ClientSession:
         '_request_class', '_response_class',
         '_ws_response_class', '_trace_configs')
 
+    _default_headers: CIMultiDict[str]
```

Comment:
aiohttp still supports Python 3.5, this syntax doesn't work.

I'm open for discussing the drop of Python 3.5 in aiohttp 4.0 though.
If you want to proceed in this direction please create a new issue for discussion.

Otherwise please use Python 3.5 compatible syntax in the PR